backend/attendance/src/main/java/org/egov/web/models/AttendanceRegisterSearchCriteria.java

```diff
@@ -44,7 +44,7 @@ public class AttendanceRegisterSearchCriteria {
     private String staffId;
 
     @JsonProperty("referenceId")
-    private String referenceId;
+    private List<String> referenceId;
```

Comment:
This will break the existing UI, add another field, which will be internal just for using the IN query.